chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/auth/UserDAO.java

```diff
@@ -251,6 +246,27 @@ private UserEntity getEntityByName(String uname) throws UserRetrievalException {
         return result.get(0);
     }
 
+    private UserEntity getEntityByNameOrEmail(String email) throws MultipleUserAccountsException, UserRetrievalException {
```

Comment:
I think the addition of this method means there are now two methods in this class with essentially the same HQL. Can you make the one that returns the DTO call this method so the HQL is only in one place?